test/e2e/defaultbackend/default_backend.go

```diff
@@ -92,7 +89,7 @@ var _ = framework.IngressNginxDescribe("[Default Backend]", func() {
 
 		f.UpdateNginxConfigMapData("enable-access-log-for-default-backend", "true")
 
-		f.HTTPTestClient().
+		f.HTTPTestClient2().
```

Comment:
Why the 2?

Reply:
just experimenting to not rename all the occurrences and still have the first lib as a control, the idea is to remove the old library later.

we can do this in one 4k line PR or with small chunks.

Reply:
to answer your question: a bad func name.